### PR TITLE
Manage post: Removing automatic rejections and declines

### PR DESCRIPTION
**Pull request by MaeveRoseveare**

### CODE REVIEW

**Comment by MaeveRoseveare** [comment]
[Design history: Reflecting the RBD and DBD process in Manage](https://trello.com/c/cFYvwCvn/275-design-history-reflecting-the-rbd-and-dbd-process-in-manage)

File: app/posts/apply-for-teacher-training/2023-06-29-improving-interview-needs-question.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+As a service design exercise, we explored some ideas for ways that different government services involved in becoming a teacher could be more joined-up.
+
+These designs are all rough sketches to illustrate the ideas.
+
+## Step by step
+
+There are a lot of different steps involved in becoming a qualified teacher, from big ones like getting accepted onto a teacher training course, to smaller ones like completing safeguarding checks and applying for funding.
+
+The Get Into Teaching website already has a [how to become a teacher page](https://getintoteaching.education.gov.uk/steps-to-become-a-teacher) which outlines some of this.
```
Comment (Kateharries) [inline app/posts/apply-for-teacher-training/2023-06-29-improving-interview-needs-question.md]: maybe 'which outlines some of these steps'

File: app/posts/manage-teacher-training-applications/2023-06-29-removing-automatic-rejections-and-automatic-declines.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  - automatic rejections
+  - offers
+ogImage:
+  src: automatic-rejections-cover.png
+  alt: "Illustration showing a list of applications with the header 'Received – make a decision'. There are 5 applications in the list with each one showing the name of the candidate, the status each application is in, and how long ago the application was received."
+---
+
+![Illustration showing a list of applications with the header 'Received – make a decision'. There are 5 applications in the list with each one showing the name of the candidate, the status each application is in, and how long ago the application was received.](automatic-rejections-cover.png)
+
+Leading up to the 2023 to 2024 teacher training recruitment cycle, we decided to [change the application process](/apply-for-teacher-training/changing-application-process/) for candidates. Most of the changes we made were in Apply for teacher training (Apply) . However, we also needed to reflect some of these changes in Manage teacher training applications (Manage).
```
Comment (Kateharries) [inline app/posts/manage-teacher-training-applications/2023-06-29-removing-automatic-rejections-and-automatic-declines.md]: Remove extra space after '(Apply)'

File: app/posts/manage-teacher-training-applications/2023-06-29-removing-automatic-rejections-and-automatic-declines.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+- applications to be rejected automatically 40 working days after a candidate had submitted
+
+- offers to be automatically declined if a candidate does not respond within 10 working days
+
+## The issue
+
+### Provider receives an application
+
+When a provider receives an application, they have  40 working days to make a decision on it. If they do not take any action, the application is rejected automatically.
```
Comment (Kateharries) [inline app/posts/manage-teacher-training-applications/2023-06-29-removing-automatic-rejections-and-automatic-declines.md]: remove extra space after 'have'

**Review by Kateharries — APPROVED** [review]

**Pull request merged**